crates/bevy_render/src/camera/camera.rs

```diff
@@ -92,19 +127,6 @@ impl Default for ExposureSettings {
     }
 }
 
-impl ExposureSettings {
-    #[inline]
-    pub fn ev100(&self) -> f32 {
-        (self.aperture_f_stops * self.aperture_f_stops / self.shutter_speed_s).log2()
```

Comment:
We could keep this but as an impl on `PhysicalCameraParameters`.

Reply:
Sure